Mathlib/LinearAlgebra/Multilinear/Basic.lean

```diff
@@ -77,14 +77,11 @@ since `_inst` is a free variable and so the equality can just be substituted.
 
 open Function Fin Set BigOperators
 
-universe u v v' v₁ v₂ v₃ w u'
-
-variable {R : Type u} {ι : Type u'} {n : ℕ} {M : Fin n.succ → Type v} {M₁ : ι → Type v₁}
-  {M₂ : Type v₂} {M₃ : Type v₃} {M' : Type v'}
+variable {R S ι : Type*} {n : ℕ} {M : Fin n.succ → Type*} {M₁ : ι → Type*} {M₂ M₃ M' : Type*}
```

Comment:
nit: I'm starting to find that
```lean
universes uR uS uι
variable {R : Type uR} {S : Type uS} {ι : Type uι}
```
e.t.c. is worth the effort, as it leads to much more comprehensible (and shorter!) error messages when type arguments are in the wrong order.